src/engine/Collision/CollisionGroupManager.ts

```diff
@@ -0,0 +1,42 @@
+import { CollisionGroup } from './CollisionGroup';
+
+export class CollisionGroupManager {
```

Comment:
Needs a comment, also this should be the recommended way to create collision groups